_posts/2020-05-15-release-2.5.0.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: Release v2.5.0
+tags: ["v2.5.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 15/05 2020 the new version 2.5.0 is out
+<!--more-->
+
+## Standardization & Conformance Improvements
+
+- [#1294]({{ site.github.issues_url }}/1294) **Record ID generation:** `enceladus_record_id` column is added in Enceladus (Standardization/Conformance) if it does not exist. By default it contains UUID for each record, but the implementation is overridable with `-Denceladus.recordId.generation.strategy={uuid|stableHashId|none}` 
+- [#1322]({{ site.github.issues_url }}/1322) **\_INFO file validation:** Added `control.info.validation` configuration option for \_INFO file validation which can have the following values: `strict` will fail the job on failed validation, `warning` will display the error as warning, but will not fail the job and `none` will ignore the result of the validation. Providing an incorrect value will fail the job with a RuntimeException
```

Comment:
maybe speak about raw and source checkpoints that are mandatory.

Reply:
Should not be in release notes IMO. Should be in documentation and referenced here.
Will try to put something together.